src/services/programService/index.js

```diff
@@ -45,6 +45,9 @@ const formatPrivateProgramList = (programList) => programList.map(formatPrivateP
 const getDataCenterByShortName = (shortName, dataCenterResponse) =>
 	dataCenterResponse.filter((dataCenterObject) => dataCenterObject.shortName === shortName);
 
+const getDataCenterById = (id, dataCenterResponse) =>
+	dataCenterResponse.filter((dataCenterObject) => dataCenterObject.id === id);
```

Comment:
are we expecting 1 data or many?
from the name I'm assuming a single data center - so `Array.find` might work better

Reply:
Ah, great call, see what you mean, will update